### DISCUSSION TITLE
when turn on printer it starts cleaning the bed

### DISCUSSION THREAD

**Discussion by sidneyns** [smartin015/continuousprint#66] [General]
Hello. 

When I start the printer and it has a queue, it starts the printer with the cleaning bed script. 

Problem is that my cleaning bed script it turns down the temperature and also cleans the bed, with a specific code when the bed is centered. 
Problem is that it cools down the bed and the worse is that since the bed was turned off, the bed is normally at lower position and when it starts to clean the bed, it makes terrible noise since it tries to clean the bed at the wrong initial position

Can someone help me on the best config?   Should I put to lower temp at the bed cooldown script? It might solve the inicial problem to cool down the bed, but it will not fix the fact that will try to clean the bed while the bed is low

M17 ;enable steppers
G91 ; Set relative for lift
G0 Z10 ; lift z by 10
G90 ;back to absolute positioning
G0 X30 Y1 F3600
M190 R30 ; set bed to 25 for cooldown
M190 R30 ; set bed to 25 for cooldown
M190 R30 ; set bed to 25 for cooldown
M190 R30 ; set bed to 25 for cooldown
G0 X30 Y235
G0 Z1
G0 X30 Y1
G0 X30 Y235
G0 X100 Y235
G0 Y0
G0 X100 Y235
G0 X160 Y235
G0 Y0
G0 X160 Y235
G0 X220 Y235
G0 Y0
G0 Y235
G28 ; home

**Comment by smartin015** [comment]
Hi Sidney - which version of CPQ are you using? 

My printer running `v1.6.0` doesn't start with the clearing script, it starts with the print I queued.

**Comment by sidneyns** [comment]
I am using 1.6.0

It starts on clearing script when you have itens still to be printed and
the button to start manage turn on.

Normally when something gets stuck in the printer and I have to restart the
queue

**Reply by smartin015** [reply]
I see. Can you update the plugin to the latest version (as of this post, v2.1.1) and try again? If it still starts with the clearing script, please open a Github issue and provide a sysinfo dump using the instructions in the issue template, so I can debug further. Thanks!
